### ISSUE TITLE
Incorrect z-scoring during cross-validation

### ISSUE THREAD

**Issue by aarmey** [meyer-lab/resistance-MS#158]
I didn't look at exactly where this is being run, but the cross-validation performed here is wrong. The normalization has to occur during the training step, and then be applied to the test data with the same stdev and mean calculated during training. [`sklearn.preprocessing.scale`](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.scale.html) helps to perform this by separating `X.fit_transform()` and `X.transform()`.

(`sklearn.cross_decomposition.PLSRegression` is supposed to handle this for you, anyway, when `scale=True`.)

https://github.com/meyer-lab/resistance-MS/blob/90d15e719a56e0fbf63ae81cb8c237470f01a971/msresist/figures/figure3.py#L208
